apollo-gradle-plugin/src/main/groovy/com/apollostack/android/gradle/ApolloIRgenTask.groovy

```diff
@@ -0,0 +1,60 @@
+package com.apollostack.android.gradle
+
+import com.google.common.collect.Sets
+import com.moowork.gradle.node.task.NodeTask
+import org.gradle.api.GradleException
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.util.PatternSet
+
+public class ApolloIRgenTask extends NodeTask {
+  private static final String APOLLO_CODEGEN = "node_modules/apollo-codegen/lib/cli.js"
+  static final String NAME = "generate%sApolloIR"
+  List<ApolloExtension> config
+  String variant
+
+  @InputFiles
+  Collection<File> getInputFiles() {
```

Comment:
The extension stores the file paths as a multimap with keys corresponding to the relative path from the project root, and values are all `grapghql` files found.
This builds a Set containing those files. 
(We can go further and have it as a Set of absolute paths since that's the only use case here).